types.go

```diff
@@ -62,6 +62,17 @@ const (
 	OrderIntentAuthorize string = "AUTHORIZE"
 )
 
+// Possible value for `status` in GetOrder
+//
+// https://developer.paypal.com/docs/api/orders/v2/#orders-get-response
+const (
+	OrderStatusCreated   string = "CREATED"
+	OrderStatusSaved     string = "SAVED"
+	OrderStatusApproved  string = "APPROVED"
+	OrderStatusVoided    string = "VOIDED"
+	OrderStatusCompleted string = "COMPLETED"
+)
```

Comment:
```suggestion
type OrderStatus string

const (
	OrderStatusCreated   OrderStatus = "CREATED"
	OrderStatusSaved     OrderStatus = "SAVED"
	OrderStatusApproved  OrderStatus = "APPROVED"
	OrderStatusVoided    OrderStatus = "VOIDED"
	OrderStatusCompleted OrderStatus = "COMPLETED"
)
```

Reply:
Of course this can be done, but I don't see why order status should be the odd one out implemented this way. Not sure it makes sense to be inconsistent with the rest of the project?

If you want this, wouldn't it be better for _all_ consts to be implemented as custom types? And wouldn't that be API breaking?

Reply:
thats fair. i'll refactor to do this across the baord and bump the version. I do have some  bigger changes that I have been meaning to do for a while